components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -124,6 +133,25 @@ public void testDefault(String suffix) throws FormatException, IOException {
     }
   }
 
+  @Test(dataProvider = "options")
+  public void testOptions(String options) throws FormatException, IOException {
+    outFile = File.createTempFile("test", ".ome.tiff");
+    outFile.delete();
```

Comment:
@bramalingam the original design comes from https://github.com/openmicroscopy/bioformats/pull/2725 where these unit tests where introduced. The idea is to generate a unique temporary file path per test without creating the file else `bfconvert` hangs.
The crudest solution I could find was to use `File.createTempFile` and then delete the file. There might be a more elegant way to do that using a third party dependency guava /cc @mtbc